docs/source/build_a_lint_rule.ipynb

```diff
@@ -121,14 +121,86 @@
     "        ..."
    ]
   },
+  {
+   "cell_type": "raw",
+   "metadata": {},
```

Comment:
You need to follow the steps in https://nbsphinx.readthedocs.io/en/0.7.1/raw-cells.html to specify the format as `rst`.
(which generates `"raw_mimetype": "text/restructuredtext"` metadata)
Otherwise the output is in wrong format.
![image](https://user-images.githubusercontent.com/3840867/92533233-debe3180-f1e6-11ea-80a7-e0c407bd96ee.png)
 
You can run `tox -e docs` locally to generate html files and verify the result in a browser.

Reply:
thank you, looking into it

Reply:
Format specified